src/test/java/io/pravega/connectors/flink/FlinkPravegaTableFactoryTest.java

```diff
@@ -0,0 +1,272 @@
+/**
```

Comment:
Maybe I missed that, but are there any kind of integration tests for the table connector that interact with an actual Pravega (standalone) instance? This would be great to validate the interactions of the connector with Pravega.

Reply:
Yes system integration tests are added and it uses InMemory cluster.